Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
+MAINTAINER Aleksandar Todorović <aleksandar@r3bl.me>
```

Comment:
I agree completely. It's just that I was not able to find someone to contact right away when I noticed that something's wrong with this pull request.